docs/components/pages/pack-home/PackBenchmarksGraph.tsx

```diff
@@ -34,7 +34,7 @@ export function BenchmarksGraph({
   const data: BenchmarkData = benchmarkData[category][numberOfModules];
   const keys = bars.map((bar) => bar.key);
   const longestTime = Math.max(...keys.map((key) => data[key])) * 1000;
-  const roundedLongestTime = Math.ceil(longestTime / 5000) * 5000 + 5000;
+  const longestTimeWithPadding = longestTime * 1.15;
```

Comment:
what does the 1.15 represent?

Reply:
Just adds a little padding at the end, based on a percentage instead of the raw value.